src/App.js

```diff
@@ -75,6 +60,13 @@ class App extends Component {
         .catch(error => console.log(error))
     }
 
+    hidePopup = (event) => {
+      event.preventDefault();
+      this.setState({
+        showPopup: false
+      });
```

Comment:
This looks great! I like the direction you took with this. Glad you came up with this idea.